packages/messaging/lib/index.js

```diff
@@ -154,12 +155,34 @@ class FirebaseMessagingModule extends FirebaseModule {
     return this.native.getIsHeadless();
   }
 
-  getToken() {
-    return this.native.getToken();
+  getToken({ projectId, senderId } = {}) {
+    if (!isUndefined(projectId) && !isString(projectId)) {
+      throw new Error("firebase.messaging().getToken(*) 'projectId' expected a string.");
+    }
+
+    if (!isUndefined(senderId) && !isString(senderId)) {
+      throw new Error("firebase.messaging().getToken(*) 'senderId' expected a string.");
+    }
+
+    return this.native.getToken(
+      projectId || this.app.options.projectId,
+      senderId || this.app.options.messagingSenderId,
+    );
   }
 
-  deleteToken() {
-    return this.native.deleteToken();
+  deleteToken({ projectId, senderId } = {}) {
+    if (!isUndefined(projectId) && !isString(projectId)) {
+      throw new Error("firebase.messaging().deleteToken(*) 'projectId' expected a string.");
+    }
+
+    if (!isUndefined(senderId) && !isString(senderId)) {
+      throw new Error("firebase.messaging().deleteToken(*) 'senderId' expected a string.");
+    }
+
+    return this.native.deleteToken(
+      projectId || this.app.options.projectId,
```

Comment:
So once it's not the projectId but the appName that the getInstance expects, you should pass this.app.name, I've tested locally here and it worked